aim/web/ui/src/components/MediaSet/MediaSet.tsx

```diff
@@ -507,6 +383,166 @@ const MediaGroupedList = React.memo(function MediaGroupedList({
 },
 areEqual);
 
+function DepthSlider({
```

Comment:
please move this component to a separate file

Reply:
done , thanks